modules/integration/tests-integration/tests-backend/src/test/resources/apiSwaggerFiles/scim2.yaml

```diff
@@ -1226,7 +1226,35 @@ paths:
           description: Schema not found
           schema:
             $ref: '#/definitions/ErrorSchemaNotFound'
+  /Schemas:
```

Comment:
```suggestion
  /schemas:
```

Reply:
This is how the API resource is implemented